pkg/sql/sem/tree/cast.go

```diff
@@ -1493,12 +1546,6 @@ func performCastWithoutPrecisionTruncation(ctx *EvalContext, d Datum, t *types.T
 			res = NewDInt(DInt(v.Unix()))
 		case *DTimestampTZ:
 			res = NewDInt(DInt(v.Unix()))
-		case *DDate:
```

Comment:
you'll need to put in a release note for this.
is this something we want to do for this commit?

Reply:
Good point. I'll do this in another PR.